client/src/ShareButton.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+  position: absolute;
+  margin: 2% 0% 0% 82%;
+  border-radius: 4px;
+  background-color: white;
+  color: #484848;
+  height: 36px;
+  width: 91px;
+`;
+
+const StyledDiv = styled.div`
+  display: flex;
+  justify-content: center;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+`;
+
+const StyledImg = styled.img`
+  width: 15px;
+  height: 15px;
+  padding: 0px 15px 0px 0px;
+  display: flex;
+  flex-direction: row;
+`;
+
+class ShareButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { shareButtonClicked: false };
+
+    this.onShareButtonClick = this.onShareButtonClick.bind(this);
+  }
+
+  onShareButtonClick(event) {
+    event.preventDefault();
+    this.setState(prevState => ({ shareButtonClicked: !prevState.shareButtonClicked }));
+  }
+
+  render() {
+    const { shareButtonClicked } = this.state;
+
+    return (
+      <StyledButton type="button" onClick={this.onShareButtonClick}>
+        <StyledDiv>
+          <StyledImg src="./../icons/share-icon-svg-5.jpg" alt="like icon" shareButtonClicked={shareButtonClicked} />
```

Comment:
The name "share-icon-svg-5.jpg" suggests this should be a svg. Should we use that instead?

Reply:
I was having trouble getting the image to show up when using an svg tag. I have a note in my trello board to revisit in the future. Good catch though!